_data/taps/versions/mixpanel.yml

```diff
@@ -2,12 +2,19 @@
 #     MIXPANEL VERSIONS      #
 # -------------------------- #
 
-latest-version: "23-12-2015"
+latest-version: "2"
 
 released-versions:
   - number: "23-12-2015"
     status: "released"
     date-released: "December 23, 2015"
     # date-last-connection:
     deprecation-date: ""
-    sunset-date: ""
\ No newline at end of file
+    sunset-date: ""
+
+  - number: "2"
```

Comment:
Normally, this would be right and it would be `2`, but because Mixpanel is being "tapified" - that is, we're making it into a Singer tap - it's actually `1`.

Any integration that's tapified should always start with `v1`. There aren't many of these left - Trello, Square, and perhaps a handful of others - so you won't encounter it for much longer. After the last integration is tapified, you won't have to think about it again :)